### DISCUSSION TITLE
How can only the owner to edit the repo. for an org

### DISCUSSION THREAD

**Discussion by SRIKAR-B-S-S** [community/community#22358] [New to GitHub]
<p>Hi Guys!<br>
I love :heart: Github</p>
<p>And I am CEO, Founder &amp; Chairman of Funmora Developers</p>
<p>I made an organization in Github but I want it to be like:</p>
<p>We have an main repo. it is a private one…</p>
<p>So I want it to be like, only I should edit that main repo. , Any of the other members of my company don’t have access to edit/view the repo. it this possible?</p>

**Comment by SRIKAR-B-S-S** [comment]
<p>please help anyone…<br>
I am in a hurry as its currently very important for us…</p>

**Comment by SRIKAR-B-S-S** [comment]
<p>Please reply anyone!</p>

**Comment by SRIKAR-B-S-S** [comment]
<p>Is anyone there who could help me?</p>

**Comment by SRIKAR-B-S-S** [comment]
<p>Is anyone there guys :sob:</p>
<p>There are <strong>33</strong> but no one could help me…</p>

**Comment by SRIKAR-B-S-S** [comment]
<p>63 Views<br>
0 replies(by others)</p>

**Comment by billythekid** [comment]
<p>Hi there,</p>
<p>We have some documentation on this that you might find helpful.</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories" target="_blank" rel="noopener">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/330170/181070514-ed3fcdda-53b7-4f2e-9441-c9a47fbcefed.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/organizations/managing-access-to-your-organizations-repositories" target="_blank" rel="noopener">Managing access to your organization's repositories - GitHub Docs</a></h3>



  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>I’d definitely recommend familiarizing yourself with (or bookmarking) the organization section if you’ll be managing an organization as there are a lot of hints and tips in there to help make this easy for you.</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/organizations">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/organizations" target="_blank" rel="noopener">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/330170/181070516-f683c4fe-00cc-48eb-92e3-dcc79c7f87a5.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/organizations" target="_blank" rel="noopener">Organizations and teams - GitHub Docs</a></h3>



  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Hope that helps.</p>
<p>Cheers,<br>
Billy :cowboy_hat_face:</p>

**Comment by SRIKAR-B-S-S** [comment]
<p>Thanks a lot <a class="mention" href="/billythekid">@billythekid</a><br>
I will check out the links and read them<br>
:grin:</p>

**Comment by m-asadullah** [comment, marked as answer]
<p>Change your members role to only “Read” and select your own role to “Admin”, so in this way only admin can get full access to repo but other can only read them or view the code to make discussions only.</p>

**Comment by SRIKAR-B-S-S** [comment]
<aside class="quote no-group" data-username="Eddiev51" data-post="10" data-topic="215513" data-full="true">
<div class="title">
<div class="quote-controls"></div>
 Eddiev51:</div>
<blockquote>
<p>Can i pay this pi to purchase automobile car or something? 0</p>
</blockquote>
</aside>
<p>Sorry I can’t get you <a class="mention" href="/eddiev51">@Eddiev51</a></p>

**Comment by SRIKAR-B-S-S** [comment]
<p>Yes, Thanks <a class="mention" href="/m-asadullah">@m-asadullah</a> :+1:</p>

**Comment by SRIKAR-B-S-S** [comment]
<p>I wish I could mark 2 posts as Solution</p>
